app/controllers/v0/benefits/document_uploads_controller.rb

```diff
@@ -0,0 +1,150 @@
+          key :consumes, %w[multipart/form-data]
+
+          parameter do
+            key :name, :metadata
+            key :description, 'Information about the user and document being submitted. Must conform to the DocumentMetadata model.'
+            key :required, true
+            key :in, :formData
+
+            schema do
+              key :'$ref', :DocumentMetadata
+            end
+          end
+          parameter do
+            key :name, :document
+            key :description, 'Document contents. Must be provided in PDF format.'
+            key :required, true
+            key :type, :file
+            key :in, :formData
+          end
+          parameter do
+            key :name, :attachment1
+            key :description, 'Optional attachment contents. Must be provided in PDF format.'
+            key :required, false
+            key :type, :file
+            key :in, :formData
+          end
+          parameter do
+            key :name, :attachment2
+            key :description, 'Optional attachment contents. Must be provided in PDF format.'
+            key :required, false
+            key :type, :file
+            key :in, :formData
+          end
+
+          security do
+            key :api_key, []
+          end
+
+          response 200 do
+            key :description, 'Upload received'
+            schema do
+              property :id, type: :string, example: 'b62e1379-e703-4aa7-a9dd-c74e66710f46', description: 'Identifier for subsequent getStatus requests'
+            end
+          end
+
+          response 400, description: 'Bad Request' do
+            key :'$ref', :DocumentUploadErrors
+          end
+
+          response 401, description: 'Unauthorized Request'
+          response 403, description: 'Bad API Token'
+          response 413, description: 'Request too large'
+        end
+      end
+
+      swagger_schema :DocumentMetadata do
+        key :required, %i[veteranFirstName veteranLastName fileNumber
+                          receiveDate zipCode source uuid docType
+                          numberPages hashV numberAttachments]
+
+        property :veteranFirstName, type: :string, example: 'Jane'
+        property :veteranLastName, type: :string, example: 'Doe'
+        property :fileNumber, type: :string, example: '999887777', description: '8 or 9 digit number'
+        property :receiveDate, type: :string, example: '2001-01-01 12:00:00', description: 'Date/time document was received in US Central Time'
+        property :zipCode, type: :string, example: '20571'
+        property :source, type: :string, example: 'MyApplication', description: 'Your document upload source identifier as defined in developer portal'
+        property :uuid, type: :string, example: 'b6571888-48b2-436d-b938-9807d8a2a4ef', description: 'A random unique identifier for this submission'
+        property :docType, type: :string, example: 'pension', description: 'Document type, one of "21-22", "526ez"'
```

Comment:
we've talked with the CSRA team about using the full form id in all caps for this field, like "22-1990E" or "21P-527EZ" instead of just "pension" as mentioned in the older documentation

Reply:
@lihanli - is that a change that's currently in place?

Reply:
we are currently sending docTypes like this, i think they are allowing any string for this field

Reply:
Awesome, we'll have to update as soon as the change goes through.